app/javascript/components/app-resume.vue

```diff
@@ -2,24 +2,32 @@
   <div class="resume">
     <div class="resume__container">
       <h3 class="resume__title">Tu Compra</h3>
-      <img class="resume__logo" src="~/assets/images/platanus_logo.svg">
+      <img
+        class="resume__logo"
+        src="~/assets/images/platanus_logo.svg"
+      >
 
       <div class="resume__erase">
-        <font-awesome-icon icon="trash-alt" @click="cleanKart(); cleanInvoice();" />
+        <font-awesome-icon icon="trash-alt"
```

Comment:
Attribute "icon" should be on a new line                      vue/max-attributes-per-line